test/test_time.cpp

```diff
@@ -72,6 +72,7 @@ static void* test_blackhole_thread(void *arg)
   return NULL;
 }
 
+#if 0
```

Comment:
Is this intended for inclusion with this PR? Or was this a local change, that wasn't supposed to be included?

CC: @catap

Reply:
as far as I understand this PR it shouldn't affect that. If it affects => somethings is very wrong

Reply:
It was intended but unrelated. That function isn't called and caused a warning to that effect when compiling tests, so I #if'd it out. I can restore the warning if you prefer.

Reply:
> It was intended but unrelated. That function isn't called and caused a warning to that effect when compiling tests, so I #if'd it out. I can restore the warning if you prefer.

No, you don't need to do that. But if it's unused, should we simply remove it?

Reply:
Sure. I thought that might be too drastic. It was added last month but not used then. I've removed it. If it needs to be brought back, it's still in the history.